scraper/smashgg.py

```diff
@@ -134,22 +136,46 @@ def get_smashgg_matches(self):
                 if loser_id is None:
                     continue
 
-                round_name = match.get("fullRoundText", None)
-                round_num = match.get("round", None)
-                best_of = match.get("bestOf", None)
+                # CHECK FOR A BYE OR A DQ
+                if loser_id is None:
+                    continue
+                else:
+                    entrant1_score = match['entrant1Score']
+                    entrant2_score = match['entrant2Score']
+                    if entrant1_score == -1 or entrant2_score == -1:
+                        continue
 
                 for prop in SET_TIME_PROPERTIES:
                     cur_time = match.get(prop, None)
                     if cur_time:
                         self.date = min(self.date, datetime.datetime.fromtimestamp(cur_time))
 
+                round_name = match.get("fullRoundText", None)
+                round_num = match.get("round", None)
+                best_of = match.get("bestOf", None)
+                try:
+                    name = set['fullRoundText']
```

Comment:
there is no set variable; i think this is also a carryover from a previous merge.